src/tests/ftest/server/daos_server_dump.py

```diff
@@ -45,8 +46,9 @@ def tearDown(self):
                 "{}: rc={}".format(val, key)
                 for key, val in ret_codes.items() if key != 0
             ]
-            print("no ULT stacks dump found on following hosts: {}".format(", ".join(failed)))
-            self._Test__status = 'FAIL'
+            self.log.info("no ULT stacks dump found on following hosts: {}".format(", ".join(failed)))
```

Comment:
(pylint-line-too-long) Line too long (102/100)
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions